app/models/User.py

```diff
@@ -118,8 +119,9 @@ def get_profile_picture(self, size=50):
         if self.image:
             return self.image.url()
         if not self.image_url:
-            return ('https://lh6.googleusercontent.com/-K9HZ5Z5vOU8/'
-                    'AAAAAAAAAAI/AAAAAAAAAAA/yRoMtBSXoxQ/s48-c/photo.jpg')
+            from app import app
```

Comment:
That definitely needs a comment otherwise some idiot like me would refactor and move this import to the top.